tools/oidc-signin-tool/src/TestBrowserAuthorizationClient.ts

```diff
@@ -138,8 +138,24 @@ export class TestBrowserAuthorizationClient implements FrontendAuthorizationClie
       };
     }
 
+    const proxyUrl = process.env.HTTPS_PROXY;

```

Comment:
@ramanujam-raman I don't see any tests using this, do we need to change the test auth client to work with a proxy? This shouldn't be used in a production environment and is only valid in a browser workflow, not a client creds flow.

Reply:
I needed to make these changes only because I was running all our integration tests with the proxy enabled - it was very useful for me to do that to resolve proxy issues. 

We should setup some permanent tests around this, but like I mentioned this is still WIP. I anticipate further changes, especially around electron to fully address this.